client-src/src/Facility/FacilityDetails.js

```diff
@@ -40,14 +46,25 @@ class FacilityDetails extends Component {
                 <nav>
                     <div class="nav-wrapper blue accent-1">
                         <ul class="left">
+                        {this.state.activePage ==="summary" ? (
```

Comment:
I forgot to show a better way of doing this... `<li className={this.state.activePage==='summary' ? 'active' : ''}>`

Reply:
Make sure to do this for all other cases